File: src/main/java/com/mall/configure/properties/MqttConfigureProperties.java
package com.mall.configure.properties;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties("mqtt")
public class MqttConfigureProperties {

    private String broker;

    private String password;

    private String username;

    private int timeout;

    private int keepalive;

    public String getBroker() {
        return broker;
    }

    public void setBroker(String broker) {
        this.broker = broker;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public int getTimeout() {
        return timeout;
    }

    public void setTimeout(int timeout) {
        this.timeout = timeout;
    }

    public int getKeepalive() {
        return keepalive;
    }

    public void setKeepalive(int keepalive) {
        this.keepalive = keepalive;
    }

}

File: src/main/java/com/mall/configure/properties/MqttConfigureProperties.kt
package com.mall.configure.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties("mqtt.order")
open class MqttOrderConfigureProperties {

    var broker: String? = null

    var password: String? = null

    var username: String? = null

    var timeout: Int = 0

    var keepalive: Int = 0

}


@Configuration
@ConfigurationProperties("mqtt.wxlite")
open class MqttWxLiteConfigureProperties {

    var broker: String? = null

    var password: String? = null

    var username: String? = null

    var timeout: Int = 0

    var keepalive: Int = 0

}